angular-client/components/app.js

```diff
@@ -16,11 +16,20 @@ app.controller('app' , function ($scope,$http ) {
 		console.log('error')
 	})
   }
-
-  			$scope.IsVisible = true;
-
-
+  var geee = function () {
```

Comment:
Variable name should be more descriptive. Reading this code, I have no idea what the function is supposed to do. When you give your variables and functions descriptive names, it helps your team members read your code.